distribution/docs/src/main/resources/content/_tables/catalog.ui.config.adoc

```diff
@@ -26,33 +26,42 @@
 |Imagery Providers
 |imageryProviders
 |String
-|List of imagery providers to use. Valid types are: OSM (OpenStreetMap), AGM (ArcGisMap), BM (BingMap), WMS (WebMapService), WMT (WebMapTile), TMS (TileMapService), GE (GoogleEarth).
+|List of imagery providers to use. Valid types are: *OSM* (OpenStreetMap), *AGM* (ArcGisMap), *BM* (BingMap), *WMS* (WebMapService), *WMT* (WebMapTile), **TMS** (TileMapService), and *GE* (GoogleEarth).
```

Comment:
There are no content changes in this cell. Only bold, italics, and monospace formatting was updated.